docs/source/create_channel/create_channel_participation.md

```diff
@@ -0,0 +1,550 @@
+
+#### Remove the system channel
+
+- Before attempting these steps ensure that you have [upgraded](../upgrade.html) your Fabric network to v2.3 or higher.
+- Modify the `orderer.yaml` for each ordering node to support this feature and restart the node. See the orderer [sampleconfig](https://github.com/hyperledger/fabric/blob/{BRANCH}/sampleconfig/orderer.yaml) for more information about these required parameters.
+    - `General.BootstrapMethod` - Set this value to `none`. Because the system channel is no longer required, the `orderer.yaml` file on each orderer needs to be configured with `BootstrapMethod: none` which means that no bootstrap block is required or used to start up the ordering service.
+    - `Admin.ListenAddress` - The orderer admin server address (host and port) that can be used by the `osnadmin` command to configure channels on the ordering service. This value should be a unique `host:port` combination to avoid conflicts.
+    - `Admin.TLS.Enabled:` - This value must be set to `true`.
+    - `Admin.TLS.PrivateKey:` - The path to and file name of the orderer private key issued by the TLS CA.
+    - `Admin.TLS.Certificate:` - The path to and file name of the orderer signed certificate issued by the TLS CA.
+    - `Admin.TLS.RootCAs:`  - The path to and file name of the orderer TLS CA Root certificate.
+    - `Admin.TLS.ClientAuthRequired:` This value must be set to `true`. Note that while mutual TLS is required for all operations on the orderer Admin endpoint, the entire network is not required to use Mutual TLS.
+    - `Admin.TLS.ClientRootCAs:` - The path to and file name of the admin client TLS CA Root certificate. In the folder structure above, this is `admin-client/client-tls-ca-cert.pem`.
+    - `ChannelParticipation.Enabled:` - Set this value to `true` to enable this feature on the orderer.
+- Restart each ordering node.
+- Put the [system channel into maintenance mode](../kafka_raft_migration.html#entry-to-maintenance-mode) using the same process for the Kafka to Raft migration. This action stops new channel creation transactions from coming in.
+- Remove the system channel from the set of orderers, one by one. During the system channel removal from an orderer, the orderer stops servicing all other application channels it is a member of until removal is complete. Therefore, care must be taken to ensure that the remaining orderers can still function and reach consensus. This operation can be staggered, such that depending on the fault tolerance setup of the respective channels, no channel down time is experienced. If an application channel can tolerate one server offline, you should still be able to submit transactions to the channel, via the other orderers that are not undergoing the removal at that time.  Use the `osnadmin channel list` command to view the channels that this orderer is a member of:
```

Comment:
> During the system channel removal from an orderer, the orderer stops servicing all other application channels it is a member of until removal is complete.

Is this true? @stephyee @tock-ibm

Reply:
Here's the comment, https://github.com/hyperledger/fabric/pull/2081#discussion_r522216983. IIUC this is true, I think we have a int test for this.